data/governments.txt

```diff
@@ -926,6 +943,7 @@ government "Quarg"
 	"death sentence" "quarg imprisonment"
 	"hostile hail" "hostile quarg"
 	"hostile disabled hail" "hostile quarg"
+	"show title"
```

Comment:
You wrote show title here instead of show title on log

Reply:
```suggestion
	"show on log"
```

Reply:
```suggestion
	"show title on log"
```